casper/src/main/resources/ListOps.rho

```diff
@@ -193,6 +193,25 @@ new  ListOps, rs(`rho:registry:insertSigned:secp256k1`), uriOut in {
     }
   } |
 
+  contract ListOps(@"any", @list, predicate, returnCh) = {
+    new combine, resultCh, stdout(`rho:io:stdout`) in {
+      ListOps!("partialFold", list, false, *combine, *resultCh) |
+      contract combine(@item, @acc, accResultCh) = {
+        new predResultCh in {
+          predicate!(item, *predResultCh) |
+          for (@isSatisfied <- predResultCh) {
+            stdout!({"item": item, "acc": acc, "isSatisfied": isSatisfied}) |
```

Comment:
Will you Command+F for appearances of `stdout` in the next changes you make to Rholang code?

Reply:
one day...